PSSM_score_Peptide/main.cpp

```diff
@@ -183,6 +186,8 @@ void getFileNamesFromArgv(int argc, char *argv[], string & PSSM_FileName, string
 		else if (string(argv[i]) == "-out") Hits_Out_FileName = string(argv[i + 1]);
 		else if (string(argv[i]) == "-NrandPSSM") numberOfRandomPSSM = size_t(atoi(argv[i + 1]));
 		else if (string (argv[i]) == "-useFactor") useFactor = true;
+		else if (string (argv[i]) == "-outputSequences") isOutputSequences=true;
+		else if (string (argv[i]) == "-sequenceHitMotifPath") sequenceHitMotifPath = string(argv[i + 1]);
```

Comment:
No validation if `outputSequences` is true and this input exists or has a valid value.
Incorrect value will cause crash when trying to write the sequences to file

Reply:
I have the check in the file - scan_peptides_vs_motifs . If the flag is not true I will not send the flag `outputSequences`.
I append a check that the use add the path for the file to print the sequences.